pkg/server/catalog/catalog.go

```diff
@@ -128,6 +130,10 @@ func Load(ctx context.Context, config Config) (_ *Repository, err error) {
 		return nil, err
 	}
 
+	_ = config.HealthChecker.AddCheck("catalog", &datastore.Health{
```

Comment:
Maybe we should name this something more granular than "catalog"?
```suggestion
	_ = config.HealthChecker.AddCheck("catalog.datastore", &datastore.Health{
```